File: spring-java/src/main/java/com/jmpark/springjava/entity/Management.java
package com.jmpark.springjava.entity;

import lombok.EqualsAndHashCode;
import lombok.Getter;
import lombok.RequiredArgsConstructor;
import lombok.ToString;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;

@Entity
@RequiredArgsConstructor
@Getter
@ToString
@EqualsAndHashCode(callSuper = false)
public class Management extends BaseTimeEntity {
    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    Integer id = 0;

    String name = null;
    String tel = null;
}

File: spring-kotlin/src/main/kotlin/com/jmpark/springkotlin/entity/Management.kt
package com.jmpark.springkotlin.entity

import javax.persistence.Entity
import javax.persistence.GeneratedValue
import javax.persistence.GenerationType
import javax.persistence.Id

@Entity
data class Management (
    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    val id : Int = 0,

    var name : String? = null,
    var tel : String? = null
) : BaseTimeEntity()